packages/office-ui-fabric-react/src/components/Calendar/Calendar.scss

```diff
@@ -694,4 +694,12 @@ $Calendar-dayPicker-margin: 10px;
 
 .singleBottomDate{
   border-radius: 0;
+}
+button {
+  background: none !important;
```

Comment:
any reason for all of these !importants? and global css rules that styles all "button" tags? (it seems like a global rule...)